python/lsst/ip/diffim/subtractImages.py

```diff
@@ -657,8 +670,8 @@ def _shapeTest(psf1, psf2):
     `bool`
         Returns True if psf1 is narrower than psf2 in either dimension.
     """
-    shape1 = getPsfFwhm(psf1, average=False)
-    shape2 = getPsfFwhm(psf2, average=False)
+    shape1 = evaluateMeanPsfFwhm(exp1, average=False, **kwargs)
+    shape2 = evaluateMeanPsfFwhm(exp2, average=False, **kwargs)
```

Comment:
Wrap the call to `getPsfFwhm` in a `try except` block, and use `evaluateMeanPsfFwhm` if there is an error.

Reply:
Then it should just test whether psfFwhm1 < psfFwhm2

Reply:
So you're saying that the evaluateMean should be the fallback and not a replacement, maybe because of performance?

Reply:
For this one check of the fwhm, yes.